### ISSUE TITLE
Screen reader is not announcing descriptive information when focus is on selected date after activating "Date required (with label)" at DatePicker with required field region

### ISSUE THREAD

**Issue by SyedRehmanRazvi** [microsoft/fluentui#11552]
User Experience:
Users who depends on screen reader are facing difficulty as screen reader is not announcing descriptive information when focus is on selected date after activating "Date required (with label)" at DatePicker with required field

Test Environment:
URL: https://fabricweb.z5.web.core.windows.net/oufr/7.76.1/#/examples/datepicker

Repro Steps:
1.Open the above URL
2.Navigate to DatePicker with required field section
3. Navigate to DatePicker with required field date field and activate it.
4. Verify whether screen reader is announcing descriptive information when focus is on selected date after activating "Date required (with label)" at DatePicker with required field

Actual Result:
Screen reader is not announcing descriptive information when focus is on selected date after activating "Date required (with label)" at DatePicker with required field. Screen reader is announcing as "Exit Table".

Expected Result:
Screen reader should announce descriptive information when focus is on selected date after activating "Date required (with label)" at DatePicker with required field. Screen reader should not announce as "Exit Table".
Please find the below snapshot
![image](https://user-images.githubusercontent.com/53210862/71415190-7a6f3800-2680-11ea-8c48-e57a88262478.png)

**Comment by micahgodbolt** [comment]
@lorejoh12 I noticed that on both datepickers(old and new) focus quickly jumps off of todays date, then 'exit table' is called, and focus returns again.

It takes a while to get there, but you can see it around 25 seconds into this gif (sadly trying to pause/start later made narrator switch focus)

![calendar](https://user-images.githubusercontent.com/1434956/71493599-c80cb180-27f4-11ea-9d43-8da938117403.gif)
